tests/app/views/test_audits.py

```diff
@@ -37,14 +47,20 @@ def add_audit_events_with_db_object(self):
             db.session.commit()
 
     def test_should_get_audit_event(self):
-        self.add_audit_events(1)
+        aid = self.add_audit_event(0)
         response = self.client.get('/audit-events')
         data = json.loads(response.get_data())
-
         assert_equal(response.status_code, 200)
-        assert_equal(len(data['auditEvents']), 1)
-        assert_equal(data['auditEvents'][0]['user'], '0')
-        assert_equal(data['auditEvents'][0]['data']['request'], 'data')
+        expected = {
+            'links': {'self': mock.ANY},
+            'type': 'supplier_update',
+            'acknowledged': False,
+            'user': '0',
+            'data': {'request': 'data'},
+            'id': aid,
+            'createdAt': mock.ANY
+        }
+        assert expected in data['auditEvents']
```

Comment:
I don't see that as being this tests job. The aim of this test is to assert that a newly created audit event is returned in the get request to the `/audit-events` end-point.
We shouldn't be assuming that the database is completely clean. We can but it'll be a quite radical overhaul of the tests. And even if we did want to assume and assert that we have a completely clean db at the start of each test then it would be better to do it somewhere else. 

I've just added a test that asserts that only one audit event is created when we're creating one. Don't want to be audit the creation of our audit events :)